packages/nuxt/src/app/plugins/revive-payload.client.ts

```diff
@@ -50,6 +50,7 @@ export default defineNuxtPlugin({
     }
     Object.assign(nuxtApp.payload, await nuxtApp.runWithContext(getNuxtClientPayload))
     // For backwards compatibility - TODO: remove later
```

Comment:
I'm still wondering *if* we have to set the payload here / why it was marked for compat only?